lib/src/widgets/help_collection/first_help_widget.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+
+/// [FirstHelpWidget] is modification of a [HelpWidget] described as first
+/// type.
```

Comment:
Let's give some useful info here:
```suggestion
/// This is a variation of the widget that is shaped as a wide and short rectange.
///
/// Best suited for layouts where you need to save some vertical space.
```

Reply:
This could also be a way to name them. Not sure calling the variations `First` and `Second` is descriptive. Instead we could do `WideShortHelpUkraineWidget`. 

What do you think? Maybe you have some ideas for this?